l10n_ir_country_states/README.rst

```diff
@@ -0,0 +1,42 @@
+Iran Country States
```

Comment:
In concrete, I miss the badge at the beginning of the file and the document title should have line before.